Lib/test/test_fcntl.py

```diff
@@ -6,7 +6,9 @@
 import struct
 import sys
 import unittest
-from test.support import verbose, cpython_only, get_pagesize
+from test.support import (
+    cpython_only, get_pagesize, is_apple, requires_subprocess, verbose
+)
```

Comment:
You seem to do this change often. In the future, could you add a trailing comma, to make future diffs nicer?
(Or simply add another line if the current one gets too long:
```
from test.support import verbose, cpython_only, get_pagesize
from test.support import is_apple, requires_subprocess
```
)

Reply:
@encukou Sure, no problem. 

One of the issues I've hit is that the "preferred" style for imports varies wildly between files. I appreciate there's 30 years of historical baggage in the existing code - but is there a reason either of those two approaches is preferred over isort/black style syntax:
```
from test.support import (
    cpython_only, 
    get_pagesize, 
    is_apple, 
    requires_subprocess, 
    verbose,
)
```
Running black over every file I touch would make for unreadable patches; but if little black touches when the line is already being altered would be looked on favourably, then that's an easy tweak to make.

Reply:
That works too -- there's a [trailing comma](https://peps.python.org/pep-0008/#when-to-use-trailing-commas), so future diffs won't need to add it.

(The 30 years of baggage also means there is no shared preference for stuff that isn't in PEP 8. I'm partial to repeating `from <name>` because it's searchable with `grep` -- you don't need a code analyzer -- but that's just personal opinion.)

> if little black touches when the line is already being altered would be looked on favourably

Sure! As long as it helps readability :)